### DISCUSSION TITLE
Typescript rewrite started

### DISCUSSION THREAD

**Discussion by doughlass** [faker-js/faker#221] [General]
While travelling I cloned this repo and started a new repo from scratch to write fake into Typescript.

Would this be something the community would like to contribute to?

So far I have been working on `en` `en_GB` and `de` basic methods involving `name` and `address` which all seem to be working but would like to open source it to have other contribute and build upon.

What worries me with fakerjs is how old it is.

**Comment by prisis** [comment]
Hey :), did you checkout this repo, its kind of 85% ported to typescript :), check the main branch out

**Reply by doughlass** [reply]
I did not know, I only found a couple of files and commits with reference to ts. Still lots to do, especially within the locales.

I'll trash my work on the project and start the locale rewrites

**Reply by Shinigami92** [reply]
See #169 :laughing:
